src/Engine/StreamStorm/ai/PydanticAI.py

```
@@ -0,0 +1,68 @@
+from pydantic_ai import Agent
+
+from .PydanticAIModelFactory import ModelFactory
+from .Base import AIBase
+from .ResponseModels import AIResponse
+
+
+
+class PydanticAI(AIBase):
+    def __init__(self, provider_name: str, model_name: str, api_key: str, base_url: str = None):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)